Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -220,6 +220,23 @@ function xASL_wrp_ResampleASL(x)
         % Apparently, subtraction was already done on the scanner/reconstruction
         fprintf('%s\n', PathPWI{iSpace});
         xASL_io_SaveNifti(PathASL{iSpace}, PathPWI{iSpace}, ASL_im, 32, false);
+	elseif x.modules.asl.bDeltaM
+		% We have subtraction images 

```

Comment:
Please be clear in the comments (like in a paper, otherwise the comments are not helpful!). You meant here that we don't have control-label images in the raw data, but only subtraction images, right?

(because otherwise, we always have subtraction images somewhere

Reply:
Good point. Thanks ;)